website/templates/project/addon/widget.mako

```diff
@@ -1,5 +1,5 @@
-<div class="panel panel-default" name="${short_name}">
-
+% if complete or 'write' in user['permissions']:
```

Comment:
I might have a misunderstanding of OSF permissions... The idea is that the widget will show if it has been completely configured and the user has permission to edit the project. Should it instead be 'admin'? Do contributors have 'admin' rights or just 'write' rights?